### PR TITLE
Issue #15957: dropped deprecated ColumnSchema properties

### PR DESCRIPTION
**Pull request by rugabarbo**
See also #15716

### CODE REVIEW

**Review by samdark — APPROVED** [review]

**Review by SilverFire — APPROVED** [review]

**Comment by SilverFire** [comment]
Thank you!

File: framework/db/mysql/ColumnSchema.php [hunk 1]
```diff
@@ -18,17 +18,6 @@
  */
 class ColumnSchema extends \yii\db\ColumnSchema
 {
-    /**
-     * @var bool whether the column schema should OMIT using JSON support feature.
-     * You can use this property to make upgrade to Yii 2.0.14 easier.
-     * Default to `false`, meaning JSON support is enabled.
-     *
-     * @since 2.0.14.1
-     * @deprecated Since 2.0.14.1 and will be removed in 2.1.
```
Comment (rob006) [inline framework/db/mysql/ColumnSchema.php]: @SilverFire How this is possible to add and deprecate property in the same release? :D

Comment (samdark) [reply framework/db/mysql/ColumnSchema.php]: Why not? :)

Comment (SilverFire) [reply framework/db/mysql/ColumnSchema.php]: Because we can :)

Comment (rugabarbo) [reply framework/db/mysql/ColumnSchema.php]: When you can

**Pull request merged**